### PR TITLE
Reword our to my

### PR DESCRIPTION
**Pull request by maximiliankolb**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)

### CODE REVIEW

**Review by adamlazik1 — APPROVED** [review]
LGTM. Careful with conflicts with #2763. I recommend waiting until it's merged.

**Comment by maximiliankolb** [comment]
> LGTM. Careful with conflicts with #2763. I recommend waiting until it's merged.

Yeah, that's how I found this "issue". Will wait, rebase, wait for GHA, and merge.

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked to 3.9: :heavy_check_mark: 
   c678d9d5e5..e11e05c96b  3.9 -> 3.9

**Pull request merged**
